proto/transaction.proto

```diff
@@ -4,13 +4,21 @@ package transaction;
 import "account.proto";
 import "digest.proto";
 
+// Represents a transaction ID.
 message TransactionId {
+    // The transaction ID.
     digest.Digest id = 1;
 }
 
+// Represents a transaction summary.
 message TransactionSummary {
+    // The transaction ID.
     TransactionId transaction_id = 1;
+
+    // The block number.
     fixed32 block_num = 2;
```

Comment:
This comment is not very helpful (i.e., we can see from the name that this is a block number). Sometimes, such things cannot be avoided, but in this case, we should actually provide more detail on what this block number means.